torchbox/torchbox.com
# Feature/navigation level one styling

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1400776561)

### Description of Changes Made

Following on from the back-end changes, this introduces a new drop-down style in the navigation that only shows the immediate children of the primary navigation items. It is smaller than the mega dropdown that shows both children and grandchildren.

It also ensures that the navigation settings are respected at mobile.

### How to Test

In the pattern library at http://localhost:8000/pattern-library/render-pattern/patterns/organisms/header/header.html

Or in your build, edit the navigation settings at http://localhost:8000/admin/settings/navigation/navigationsettings/1/ and then see the changes reflected in the build.

We are not expecting to have drop-downs in the last 2 menu items, but I have added a right-aligned style for these just in case, so that they do not disappear off the edge of the screen.

### Screenshots

<details>
  <summary>Expand to see more</summary>
![Screenshot 2024-03-08 at 13 14 15](https://github.com/torchbox/torchbox.com/assets/771869/5c741317-ce76-4d8d-9b91-1c77f2dc246f)

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
